### DISCUSSION TITLE
IP cores

### DISCUSSION THREAD

**Discussion by chim20air** [pyuvm/pyuvm#103] [Q&A] [answered]
Hi,
I was wondering how to include IP cores that are vendor specifics, something like uPROM vendor specific. I was trying to find how to compile them, and which files i need to do this kind of operations, but i couldn't find anything on this
Thanks for your time

**Comment by raysalemi** [comment, marked as answer]
> Hi,
> 
> I was wondering how to include IP cores that are vendor specifics, something like uPROM vendor specific. I was trying to find how to compile them, and which files i need to do this kind of operations, but i couldn't find anything on this
> 
> Thanks for your time

This is a good question for the **cocotb** issues, maybe on Gitter.  Once the IP is connected to **cocotb**, I can show you how to talks to it with **pyuvm**.
